extensions/amp-story/0.1/amp-story-auto-ads.js

```diff
@@ -70,6 +86,9 @@ export class AmpStoryAutoAds extends AMP.BaseElement {
 
     /** @private {Object<string, string>} */
     this.config_ = {};
+
+    /** @private {?Element} */
+    this.currentAdElement_ = null;
```

Comment:
isn't the current ad element always the last in the array?

Reply:
that array is actually the page elements. I also need a pointer to the ad element